src/base/math/interpolation.h

```diff
@@ -10,7 +10,7 @@ template <typename T>
 T interpolate(const T &op0, const T &op1, double factor)
 {
 	if (factor <= 0.0) return op0;
```

Comment:
If the `factor >= 1.0` branch need `T() * 0.0`, then here isn't it necessary the `T() * 0.0` ?

Reply:
It is not needed, but I will revert this anyway, see the other comment.